### ISSUE TITLE
Magento 2.1.0: db fields with mysql default value CURRENT_TIMESTAMP cannot be updated through the EntityManager

### ISSUE THREAD

**Issue by heldchen** [magento/magento2#5385]
## Steps to reproduce
1. Install Magento from `develop` branch.
2. load a CMS Page through `$model = $pageRepository->getById(1);`
3. modify the last update timestamp: `$model->setUpdateTime('2016-09-01 00:00:00');`
4. save through `$pageRepository->save($model);`
## Expected result
1. the `cms_page`database column `update_time` of entity 1 contains `2016-09-01 00:00:00`
## Actual result
1. the database column "update_time" contains the old value

the reason being in `Magento\Framework\EntityManager\Db\UpdateRow:prepareData`:

```
        foreach ($connection->describeTable($metadata->getEntityTable()) as $column) {
            if ($column['DEFAULT'] == 'CURRENT_TIMESTAMP' || $column['IDENTITY']) {
                continue;
            }
```

as the column update_time has the default value `CURRENT_TIMESTAMP`, the field is not appended to the `$output` array.

please note: i'm aware that `cms_page.update_time` is auto-updated through mysql's "on update" attribute, I'm merely using this existing model as an example that can be easily reproduced without having to write tons of code.

**Comment by SerhiyShkolyarenko** [comment]
@heldchen thank you for reporting!
We created internal issue MAGETWO-58479.

**Comment by veloraven** [comment]
This issue was already fixed for develop.
Internal ticket for ver. 2.1 - MAGETWO-58514

**Comment by magento-engcom-team** [comment]
@heldchen, thank you for your report.
The issue is already fixed in 2.2.0
